mobile/library/common/network/BUILD

```diff
@@ -64,7 +64,7 @@ envoy_cc_library(
     ],
 )
 
-cc_library(
```

Comment:
I suspect that you will need to add `repository = "@envoy"` to this macro call now